src/templates/job_widget.html

```diff
@@ -0,0 +1,243 @@
+                    <div class="card-header do-card-header">
+                        <h2 class="card-title do-card-title">{{ card_title }}</h2>
+                        <ul class="nav nav-tabs card-header-tabs" role="tablist" id="jobTabs-{{ unique_id }}">
+                            <li class="nav-item" role="presentation"><button class="nav-link active" data-do-tab="status" type="button" role="tab" aria-selected="true">Status</button></li>
+                            <li class="nav-item" role="presentation"><button class="nav-link" data-do-tab="details" type="button" role="tab" aria-selected="false">Details</button></li>
+                            <li class="nav-item" role="presentation"><button class="nav-link" data-do-tab="inputs" type="button" role="tab" aria-selected="false">Inputs</button></li>
+                            <li class="nav-item" role="presentation"><button class="nav-link" data-do-tab="outputs" type="button" role="tab" aria-selected="false">Outputs</button></li>
+                            <li class="nav-item" role="presentation"><button class="nav-link" data-do-tab="raw" type="button" role="tab" aria-selected="false">Raw Progress Reports</button></li>
+                        </ul>
+                    </div>
+                    <div class="card-body">
+                        <div class="do-tabpanel tab-pane active show" data-do-panel="status">
+                            {{ status_html|safe }}
+                        </div>
+                        <div class="do-tabpanel tab-pane" data-do-panel="details">
+                            <table class="table table-bordered table-striped mb-0">
+                                <thead>
+                                    <tr>
+                                        <th>executionID</th>
+                                        <th>resourceID</th>
+                                        <th>Status</th>
+                                        <th>Started At</th>
+                                        <th>Running Time</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    {% for i in range(job_ids|length) %}
+                                    <tr>
+                                        <td>{{ job_ids[i] }}</td>
+                                        <td>{{ resource_ids[i] }}</td>
+                                        <td>{{ statuses[i] }}</td>
+                                        <td>{{ started_at[i] }}</td>
+                                        <td>{{ running_time[i] }}</td>
+                                    </tr>
+                                    {% endfor %}
+                                </tbody>
+                            </table>
+                        </div>
+                        <div class="do-tabpanel tab-pane" data-do-panel="inputs">
+                            <pre class="do-pre">{{ inputs_json|safe }}</pre>
```

Comment:
Consider using `lang="json"`

Reply:
i don't think this is supported by jinja